src/safeds_runner/server/main.py

```diff
@@ -194,12 +14,13 @@ def start_server(port: int) -> None:  # pragma: no cover
 
     logging.getLogger().setLevel(logging.DEBUG)
     # Startup early, so our multiprocessing setup works
+    app_pipeline_manager = PipelineManager()
     app_pipeline_manager.startup()
     from gevent.monkey import patch_all
-    from gevent.pywsgi import WSGIServer
 
     # Patch WebSockets to work in parallel
     patch_all()
-    logging.info("Starting Safe-DS Runner on port %s", str(port))
-    # Only bind to host=127.0.0.1. Connections from other devices should not be accepted
-    WSGIServer(("127.0.0.1", port), app, spawn=8).serve_forever()
+
+    from safeds_runner.server.server import SafeDsServer
```

Comment:
Better move all imports to the top of the file, unless this causes circular imports. IDEs can handle this better than dynamic imports. For this specific one, you can also use a relative import (`from ._server.py`).

Reply:
This would cause the warning to reappear, as the import needs to happen after the monkey-patching (as the server module imports all the http/ws dependencies).